python/llm/example/CPU/Application/hf-agent/README.md

```diff
@@ -0,0 +1,81 @@
+# BigDL-LLM Transformers INT4 Optimization for HuggingFace Transformers Agent
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on HuggingFace Transformers Agent. For illustration purposes, we utilize the [lmsys/vicuna-7b-v1.5](https://huggingface.co/lmsys/vicuna-7b-v1.5) as the reference model.
```

Comment:
In this example, we apply low-bit optimizations to [HuggingFace Transformers Agents](https://huggingface.co/docs/transformers/transformers_agents) using BigDL-LLM, which allows LLMs to use tools such as image generation, image captioning, text summarization, etc.

Reply:
Done. Thanks!